src/net64/emulator/m64plus.hpp

```diff
@@ -76,12 +76,20 @@ struct PluginInfo
 /**
  * Dynamically loaded Mupen64Plus core
  */
-struct Core
+class Core final
```

Comment:
`Core` is now private and friends with `Mupen64Plus` and since you used the default visibility of `struct` I'm using the default visibility of `class`